include/grpcpp/security/server_credentials.h

```diff
@@ -64,6 +64,13 @@ namespace experimental {
 /// Builds Xds ServerCredentials given fallback credentials
 std::shared_ptr<ServerCredentials> XdsServerCredentials(
     const std::shared_ptr<ServerCredentials>& fallback_credentials);
+
+/// Builds Binder ServerCredentials.
+///
+/// Calling \a ServerBuilder::AddListeningPort() with Binder ServerCredentials
+/// in a non-Android environment will make the subsequent call to
+/// \a ServerBuilder::BuildAndStart() returns a null pointer.
+std::shared_ptr<ServerCredentials> BinderServerCredentials();
```

Comment:
Public API's will need a gRFC.

Reply:
We decided to temporarily move the APIs back to our own namespace `grpc_binder` because we need this PR in the internal testings and the RFC process is likely to take longer than desired. We're drafting a gRFC, and will make the APIs public once the RFC is approved. Thanks!